src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -38,6 +39,23 @@ const decideDropCap = (html: string) => {
     return isLetter(first) && first;
 };
 
+const sanitiserOptions = {
+    // Defaults: https://www.npmjs.com/package/sanitize-html#what-are-the-default-options
+    allowedTags: false, // Leave tags from CAPI alone
+    allowedAttributes: false, // Leave attributes from CAPI alone
+    transformTags: {
+        a: (tagName: string, attribs: {}) => ({
+            tagName, // Just return anchors as is
+            attribs: {
+                ...attribs, // Merge into the existing attributes
+                ...{
+                    'data-link-name': 'in body link', // Add the data-link-name for Ophan to anchors
+                },
+            },
+        }),
+    },
+};
+
```

Comment:
Not sure what the scopes or use cases here are but could this live in the `sanitise-html` lib? Or will the options be different each time? (Assuming there will be next times)

Reply:
Or, maybe we could have base options defined at definition and extend them by spreading locally at use?

Reply:
> Would it be possible to replace `escapeData.tsx` with this lib?

Yeah maybe! Will put a card in to review.

> Or will the options be different each time?

I think it's likely it will be sufficiently different each time that base options wouldn't be that useful. This is basically giving us all the info (work on `a`, keep the attributes, add `data-link-name`) local to the transformation which I guess is our preferred approach now.